src/net.cpp

```diff
@@ -1228,6 +1227,9 @@ bool CConnman::AddConnection(const std::string& address, ConnectionType conn_typ
     // no limit for ADDR_FETCH because -seednode has no limit either
     case ConnectionType::ADDR_FETCH:
         break;
+    // no limit for FEELER connections since they're short-lived
+    case ConnectionType::FEELER:
```

Comment:
nit: there are several comments that describe the possible connection types and could be updated for feelers, I hope I caught them all [here](https://github.com/bitcoin/bitcoin/pull/22096/commits/c34ad3309f93979b274a37de013502b05d25fad8)

Reply:
Thanks. Updated!